README.md

```diff
@@ -64,83 +72,69 @@ class TestExampleService(NioServiceTestCase):
         })
 ```
 
-
 Each test class can only contain unit tests for one service. These unit tests are not meant for testing interaction between services. Testing interactions between services would be integration testing.
 
-### *service_name* class attribute
-
-The very first thing you need to do is set the class attribute *service_name* to your service name. This is how the test will know which service and blocks to load and configure. Above this is set to "ExampleService".
-
-### *subcriber_topics* and *pubisher_topics*
+**Set `service_name` Class Attribute**<br>The very first thing to do is change the class attribute `service_name` from "ExampleService" to your service name. This is how the test will know which service and blocks to load and configure.
 
-If the service has subscriber or publisher blocks, override these methods to return a list of the topic names in your service. This allows your tests to publish test signals to the subscribers and to assert against the published signals from the service.
+**Override `subscriber_topics` and `publisher_topics`**<br>If the service has subscriber or publisher blocks, override these methods to return a list of the topic names in your service. This allows your tests to publish test signals to the subscribers and to assert against the published signals from the service.
```

Comment:
That is the best format for a definition list I have found in markdown. I have been using it throughout. If we find something better, I'm all ears!